Code/Framework/AzFramework/AzFramework/Viewport/ScreenGeometry.h

```diff
@@ -131,21 +139,35 @@ namespace AzFramework
         return !operator==(lhs, rhs);
     }
 
+    inline ScreenVector& operator*=(ScreenVector& lhs, const float rhs)
+    {
+        lhs.m_x = AZStd::lround(aznumeric_cast<float>(lhs.m_x) * rhs);
+        lhs.m_y = AZStd::lround(aznumeric_cast<float>(lhs.m_y) * rhs);
```

Comment:
> 
> 
> This review contains many changes that is code clean up, mixed with the actual code changes.
> 
> This makes it more difficult to review since the reviewer has to do the mental exercise of going through all the code and separate what's clean up and what's not, making the PR less focus and unnecessarily larger.
> 
> In the future I'd recommend when possible to make two different PRs, it'd help to review the code faster.

Yes apologies I did fix up a couple of other things as I ran into them... it's always a tradeoff between the time it takes to separate everything and just doing the fix to improve things... My rationale was that things were _close enough_ and I was already touching some of those files, that it would be okay, but I do appreciate the frustration as a reviewer 😞 I'll see if I can split out the `ScreenGeometry.h` changes (I think I can do this with a `git reset --soft` approach). Do you have much experience with this when splitting things up? I'll make the updates you suggested too shortly

Reply:
On this particular issue I'd forgotten about the annoying `long long` and `long`, I can add a cast to be 100% sure

Reply:
>I'll see if I can split out the ScreenGeometry.h changes

I wouldn't worry about this PR. Just something to keep in mind for the future.

> Do you have much experience with this when splitting things up?

No one rule to fit all here, each set of work can have different dependencies. But maybe avoid doing the clean up in the same branch as the actual work, create a new branch instead to do the clean up, it should get approved quickly since it's trivial, get it merged, then in the first branch would just be merge the latest.

Reply:
Thanks @moraaar, I'm going to do a quick local experiment to see how easy it might be to do in future (if I inadvertently get myself into this mess again 🙈). This looks like a promising approach - https://stackoverflow.com/questions/30768148/git-split-pull-request-into-smaller-prs-based-upon-the-new-directories-in-the